tools/README-generator/make_readme.py

```diff
@@ -53,6 +60,15 @@ def generate_READMEs(app_path: str):
         else:
             disclaimer = None
 
+        # TODO: Add url to the documentation... and actually create that documentation :D
+        antifeatures = { a: antifeatures_list[a] for a in from_catalog.get('antifeatures', [])}
+        for k, v in antifeatures:
+            v['title'] = v['title'].get(lang_suffix, 'en')
+            if manifest.get("antifeatures", {}).get(k, 'en'):
+                v['description'] = manifest.get("antifeatures", {}).get(k, 'en')
```

Comment:
```suggestion
            if manifest.get("antifeatures", {}).get(k):
                v['description'] = manifest.get("antifeatures", {}).get(k).get(lang_suffix, 'en')
```

Maybe a typo here ? Not 100% sure

Reply:
In fact `.get(lang_suffix, 'en')` is probably not what we want ... we probably want a small helper similar to [this one](https://github.com/YunoHost/yunohost/blob/dev/src/utils/i18n.py#L24) : 

```python
def value_for_lang(values, lang):
     if not isinstance(values, dict):
         return values
     if lang in values:
         return values[lang]
     elif "en" in values:
         return values["en"]
     else:
         return list(values.values())[0]
  ```